layouts/joomla/form/field/textarea.php

```diff
@@ -42,12 +46,23 @@
  * @var   array    $options         Options available for this field.
  * @var   array    $inputType       Options available for this field.
  * @var   string   $accept          File types that are accepted.
+ * @var   boolean  $charcounter     Does this field support a character counter?
  */
 
+ // Initialize some field attributes.
+if ($charcounter)
+{
+	// Load the js file
+	HTMLHelper::_('script', 'vendor/short-and-sweet/short-and-sweet.js', ['version' => 'auto', 'relative' => true], ['defer' => true]);
+	// Set the css class to be used as the trigger
+	$charcounter = ' charcount';
+	// Set the text
+	$counterlabel = 'data-counter-label="' . Text::_('COM_CONFIG_METADESC_COUNTER') . '"';
+}
 $attributes = array(
 	$columns ?: '',
 	$rows ?: '',
-	!empty($class) ? 'class="form-control ' . $class . '"' : 'class="form-control"',
+	!empty($class) ? 'class="form-control ' . $class . $charcounter . '"' : 'class="form-control ' . $charcounter . '"',
```

Comment:
```suggestion
	!empty($class) ? 'class="form-control ' . $class . $charcounter . '"' : 'class="form-control' . $charcounter . '"',
```
I think there is still this space which can be removed now because otherwise double space between "form-control" and "charcount" in the html. But that's a minor thing of course.

Reply:
missed that one